.github/workflows/main.yml

```
@@ -97,7 +97,7 @@ jobs:
           ZULIP_API_TOKEN: ${{ secrets.ZULIP_API_TOKEN }}
           ZULIP_USERNAME: ${{ secrets.ZULIP_USERNAME }}
         run: |
-          cargo run sync apply --team-json build
+          RUST_LOG=sync_team=debug cargo run sync apply --team-json build
```

```suggestion
          RUST_LOG="sync_team=debug" cargo run sync apply --team-json build
```
this seems a bit cleaner

---

Changed :+1: